apple-pie/styles/style.css

```diff
@@ -0,0 +1,95 @@
+body {
+    display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+	justify-content: center;
+    align-items: center;
+}
+
+header{
+    
+ 
+    background-image: url(../images/apple-pie.jpg);
+ background-size: cover;
+ background-repeat: no-repeat;
+ 
+ color:white;
+ font-size: 100px;
+
+ width:1000px;
+ height:300px;
+ display:flex;
+ justify-content: center;
+ 
+
+}
+
+.recipe-story{
+    display:flex;
+    justify-content: center;
+
+
+}
+
+.bar {
+    width:50%;
+   display: flex;
+   justify-content: center;
+	align-items: center;
+}
+
+.bar1{
```

Comment:
when you are repeating exactly the same code for a different class, it is enough to use this class twice in a html